nltk/test/wordnet.doctest

```diff
@@ -716,6 +720,49 @@ cycles are mentioned in the output, together with the level where they occur:
       "Cycle(Synset('restricted.a.01'),-3,...)"]]
 
 
+2.1 Better solution: mst()
+..........................
+
+A Minimum Spanning Tree (MST) spans all the nodes of a relation graph once,
+while guaranteeing that each node is reached through the shortest path possible.
+In unweighted relation graphs like WordNet, a MST can be computed very
+efficiently using Breadth-First Search (BFS). Like acyclic_tree(), the new
+"unweighted_minimum_cost_spanning_tree()" function (imported in the Wordnet
+module as "mst") handles untractable trees, as the example discussed above:
+"wn.synset('concrete.a.01').mst(lambda s:s.also_sees())".
+
+But, while the also_sees() acyclic_tree of 'bound.a.01' reaches 
+'classified.a.02' through four links, using depth-first search as seen above
+(bound.a.01 > unfree.a.02 > confined.a.02 > restricted.a.01 > classified.a.02),
+in the following MST, the path to 'classified.a.02' is the shortest possible,
+consisting only in three links (bound.a.01 > unfree.a.02 > restricted.a.01 > 
+classified.a.02):
+
+    >>> pprint(wn.synset('bound.a.01').mst(lambda s:s.also_sees()))
+    [Synset('bound.a.01'),
+     [Synset('unfree.a.02'),
+      [Synset('confined.a.02')],
+      [Synset('dependent.a.01')],
+      [Synset('restricted.a.01'), [Synset('classified.a.02')]]]]
+
+
+Here also, specifying the "cut_mark" parameter increases verbosity, so that the
+cycles are mentioned in the output, together with the level where they occur:
```

Comment:
@ekaf can you explain the motivation for this please?

Reply:
Not really: the increased verbosity is nice to have in the normal "tree" function, but the motivation for computing a MST is when a tree is known to be intractable, or to include many cycles, in which case one would prefer to only output the trimmed version, So the additional parameters were indeed superfluous and have now been removed.